src/csharp/Extensions/Microsoft.Spark.Extensions.Delta.E2ETest/DeltaTableTests.cs

```diff
@@ -10,6 +10,7 @@
 using Microsoft.Spark.Extensions.Delta.Tables;
 using Microsoft.Spark.Sql;
 using Microsoft.Spark.Sql.Streaming;
+using Microsoft.Spark.Sql.Types;
 using Xunit;
 
 namespace Microsoft.Spark.Extensions.Delta.E2ETest
```

Comment:
We need to modify `DeltaFixture` to call spark with Delta 0.5.0.

Reply:
Sure, changed.